Provis.Core/Services/WorkspaceService.cs

```diff
@@ -68,6 +68,24 @@ public async Task CreateWorkspaceAsync(WorkspaceCreateDTO workspaceDTO, string u
 
             await Task.CompletedTask;
         }
+        public async Task UpdateWorkspaceAsync(WorkspaceUpdateDTO workspaceUpdateDTO, string userId)
+        {
+            var user = await _userManager.FindByIdAsync(userId);
+
+            _ = user ?? throw new HttpException(System.Net.HttpStatusCode.NotFound, "User with Id not exist");
+            
+            var workspaceRec = await _workspaceRepository.GetByKeyAsync(workspaceUpdateDTO.WorkspaceId);
+
+            _ = workspaceRec ?? throw new HttpException(System.Net.HttpStatusCode.NotFound, "Workspace with with Id not found");
```

Comment:
Length should be low then 100 characters

Reply:
Is this comment about exactly this part of code or maybe about similar to update validation, create validation code?

Reply:
79 line contains more then 100 characters please split it to two lines by , as separator